src/udev/net/link-config.c

```diff
@@ -1314,6 +1361,38 @@ int config_parse_wol_password(
         return 0;
 }
 
+int config_parse_rps_cpu_mask(
+                const char *unit,
+                const char *filename,
+                unsigned line,
+                const char *section,
+                unsigned section_line,
+                const char *lvalue,
+                int ltype,
+                const char *rvalue,
+                void *data,
+                void *userdata) {
+
+        LinkConfig *config = ASSERT_PTR(userdata);
+        int r;
+
+        assert(filename);
+        assert(lvalue);
+        assert(rvalue);
+
+        config->set_rps_cpu_mask = true;
+
+        if (streq(rvalue, "all")) {
```

Comment:
Hmm... If the setting is not specified, RPS is unchanged, and the kernel default is that RPS is disabled.
If `all` is specified, RPS is enabled with all CPUs. But, that's mostly equivalent to disabling RPS, unless CPUs will be hot plugged later. So, setting `all` is mostly meaningless.
Do you have any specific usecases of `all`? If not, please drop it.

Reply:
> If `all` is specified, RPS is enabled with all CPUs. But, that's mostly equivalent to disabling RPS.

I think that wouldn't be equivalent, from the kernel docs, suggested that enabling all CPUs is a valid use case.

> For a single queue device, a typical RPS configuration would be to set the rps_cpus to the CPUs in the same memory domain of the interrupting CPU. If NUMA locality is not an issue, this could also be **all CPUs** in the system. 

> Do you have any specific usecases of `all`?

We use RPS to prevent swapped frame caused by OOO (out of order) in our devices with SMP CPUs (4 core).

Reply:
Interesting.